config/install/field.field.media.image.field_image_storage.yml

```diff
@@ -0,0 +1,27 @@
+uuid: 1a7e8c9e-d18f-42a6-8e7c-632c9dde5aaf
+langcode: en
+status: true
+dependencies:
```

Comment:
Oh, and I forgot.  You need to add a block that looks like this
```
enforced:
  module:
    - islandora_image
```
to the dependencies section of each of your exported yaml files.  This tells drupal to uninstall the config when you uninstall the module.  It makes testing a whole lot easier, too.

Here's an example from the `islandora` repo: https://github.com/Islandora-CLAW/islandora/blob/8.x-1.x/config/install/islandora.fedora_resource_type.rdf_source.yml#L5-L7